Warning: Python module not loaded, you already have Python loaded via conda init
2022-11-17 10:42:17,000:INFO:mapsims:Parsing configuration from common.toml, co_low.toml
2022-11-17 10:42:17,128:INFO:mapsims:Nside: 4096, Modeling Nside: 4096, Ellmax: 10240
2022-11-17 10:42:17,151:INFO:mapsims:Initializing the PySM Sky object
2022-11-17 10:42:17,156:INFO:pysm3:Access data from /global/project/projectdirs/cmb/www/pysm-data/co/HFI_CompMap_CO-Type1_2048_R2.00_ring.fits
2022-11-17 10:42:17,158:INFO:pysm3:Access data from /global/project/projectdirs/cmb/www/pysm-data/co/HFI_CompMap_CO-Type1_2048_R2.00_ring.fits
2022-11-17 10:51:21,435:INFO:mapsims:Processing channel Channel MFL1
2022-11-17 10:51:21,436:INFO:mapsims:Bandpass integration for Channel MFL1
2022-11-17 10:51:39,318:INFO:mapsims:Smoothing and coord-transform for Channel MFL1
/global/common/software/cmb/zonca/conda/pycmbprod/lib/python3.10/site-packages/healpy/sphtfunc.py:414: RuntimeWarning: invalid value encountered in double_scalars
  return realalm2alm(res[0]), res[3] / norm(maps.reshape((-1,))), res[2]
2022-11-17 11:03:48,657:INFO:pysm3:Used map2alm_lsq, converged in 0 iterations, residual relative error nan
2022-11-17 11:23:12,184:INFO:mapsims:Writing output map cmbs4_co_low_uKCMB_LAT-MFL1_nside4096_0000.fits

real	43m18.891s
user	1919m28.423s
sys	6m25.513s
